### PR TITLE
Order unit tests

### PR DESCRIPTION
**Pull request by vnayakg**

### CODE REVIEW

File: src/main/kotlin/com/esops/service/UserService.kt [hunk 1]
```diff
@@ -265,7 +264,7 @@ class UserService {
 //        val timeDiffInDays: Int = ((currentTime - i.addedAt) / nanoSecsIn1Day).toInt()
             val timeDiffInSecs: Int = ((currentTime - i.addedAt) / milliSecsIn1Sec).toInt()
             for (day in 0 until i.dividedInventory.size) {
-                if ((day + 1) * duration < timeDiffInSecs) {
+                if ((day + 1) * duration <= timeDiffInSecs) {
```
Comment (anirudhr95) [inline src/main/kotlin/com/esops/service/UserService.kt:260]: This cannot be an if condition. It is unreadable. What does (day + 1) * Duration mean. What does this if do?

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -8,11 +8,11 @@ netty:
...
 vesting:
   breakup:
-    - 0.3
-    - 0.2
-    - 0.1
-    - 0.4
-  duration: 10
+    - 1
+    - 0
+    - 0
```
Comment (anirudhr95) [inline src/main/resources/application.yml:13]: This is not how you store values in YAML. How are they in an array?

File: src/test/kotlin/com/esops/order/OrderTest.kt [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import jakarta.inject.Inject
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.math.BigInteger
+
+@MicronautTest
+class OrderTest {
```
Comment (anirudhr95) [inline src/test/kotlin/com/esops/order/OrderTest.kt:20]: Service tests are not micronaut tests. They need to be unit tests. You are doing something wrong if they are micronaut tests. 

It does not require a server, why do you have this annotation.

File: src/test/resources/application-test.yml [hunk 4]
```diff
@@ -8,11 +8,11 @@ netty:
...
 vesting:
   breakup:
-    - 0.3
-    - 0.2
-    - 0.1
-    - 0.4
-  duration: 10
+    - 1
+    - 0
+    - 0
```
Comment (anirudhr95) [inline src/test/resources/application-test.yml:13]: same comment as the other yaml

File: src/test/kotlin/com/esops/order/OrderTest.kt (hunk 3 above)
Comment (yash-0007) [reply src/test/kotlin/com/esops/order/OrderTest.kt:20]: Changed injection type

**Pull request merged**